cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/ElasticsearchConn.scala

```diff
@@ -0,0 +1,267 @@
+package edu.uci.ics.cloudberry.zion.model.impl
+
+import edu.uci.ics.cloudberry.zion.model.datastore.IDataConn
+import play.api.Logger
+import play.api.libs.json._
+import play.api.libs.ws.{WSClient, WSResponse}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration.Duration
+
+class ElasticsearchConn(url: String, wSClient: WSClient)(implicit ec: ExecutionContext) extends IDataConn {
+
+    import ElasticsearchConn._
```

Comment:
I noticed in this file, the indentation is 4 space, but in our codebase other scala files are indented with two spaces. It should be an easy fix using idle like intellij.

Reply:
Sure, I will change indentation to 2 spaces.

Reply:
Indentation changed to 2 spaces.